internal/runfiles/BUILD.bazel

```diff
@@ -0,0 +1,23 @@
+load("//internal/js_library:js_library.bzl", "js_library")
+
+exports_files(
+    ["index.js"],
+)
+
+js_library(
+    name = "runfiles_helper",
+    srcs = ["index.js"],
+    module_name = "@bazel/runfiles",
```

Comment:
this will cause the linker to lay it out in the node_modules tree, right? we don't rely on custom patched resolve to make this load?